include/ignition/gazebo/PhysicsEvents.hh

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
I think we discussed putting this in a separate directory (https://github.com/ignitionrobotics/ign-gazebo/pull/869#discussion_r678725516). Do you mind doing that?

Reply:
You're right, I've mistakenly thought I already did that. I moved to the `include/gazebo/physics` directory and renamed from `PhysicsEvents.hh` to `Events.hh` (following the same pattern from the `rendering` subdir; however, `gui` has `GuiEvents.hh`, so maybe the pattern isn't that uniform...).